src/components/SendTransaction/DeployDataRequest.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <el-dialog
+    v-model="dialogVisible"
```

Comment:
Are we modifing the value of `dialogVisible`? If not, we can just passs a `value` prop
```suggestion
    :value="true"
```